samples/translate_from_mic.js

```diff
@@ -120,13 +141,13 @@ function main(encoding, sampleRateHertz, sourceLanguage, targetLanguage) {
           audioContent: chunk.toString('base64'),
         };
         stream.write(request);
+      })
+      .on('close', () => {
+        doTranslationLoop();
```

Comment:
what is the point of this recursion, is it allowing someone to say, and translate, a few words in a row?

Reply:
Something like that. It's to simulate one side of a conversation. The user can say something into their microphone, see it translated, (have the stream close), and then initiate another translation session.

This approximates some of the use cases that customers are looking for (real-time conversation translation).